LayoutTests/http/tests/eventsource/eventsource-url-attribute.html

```diff
@@ -7,14 +7,12 @@
 <body>
 <script>
 
-description("Both .URL and .url should work (for compatibility reasons).");
+description("Only .url should work, previously supported .URL should not.");
 
 var url = "http://127.0.0.1:8000/eventsource/resources/event-stream.py";
 var source = new EventSource(url);
 
-shouldBeEqualToString("source.URL", url);
 shouldBeEqualToString("source.url", url);
-shouldBeTrue("source.URL === source.url");
```

Comment:
We should probably assert `shouldBeUndefined("source.URL")`

Reply:
Done.